src/main/java/carracing/domain/Car.java

```diff
@@ -0,0 +1,33 @@
+package carracing.domain;
+
+
+public class Car {
+
+    private Name name;
+    private Position position;
+
+    public Car(String name) {
+        this.name = new Name(name);
+        this.position = new Position();
+    }
+
+    public void move(int randomNumber) {
+        this.position.getPosition(randomNumber);
+    }
```

Comment:
```suggestion
    public void move(MoveStrategy moveStrategy) {
        if (moveStrategy) {
            position = new Position(getPosition() + 1);
        }
    }
```
이렇게 전략패턴으로 표현할 수 있도록 리팩토링 해보면 어떨까요?